integration-test/src/main/scala/com/cloudera/livy/test/framework/MiniCluster.scala

```diff
@@ -176,7 +176,7 @@ object MiniLivyMain extends MiniClusterBase {
     // server. Do it atomically since it's used by MiniCluster to detect when the Livy server
     // is up and ready.
     eventually(timeout(30 seconds), interval(1 second)) {
-      val serverUrlConf = Map("livy.server.serverUrl" -> server.serverUrl())
```

Comment:
I left them as is based on @alex-the-man comment, but I agree that we should probable move these into `LivyConf`. I'll create a enum for them.